joern-cli/frontends/javasrc2cpg/build.sbt

```diff
@@ -21,3 +23,35 @@ scalacOptions ++= Seq(
 enablePlugins(JavaAppPackaging, LauncherJarPlugin)
 trapExit                      := false
 Global / onChangedBuildSource := ReloadOnSourceChanges
+
+lazy val packTestCode = taskKey[Unit]("Packs test code for JarTypeReader into jars.")
+packTestCode := {
+  import better.files._
+  import net.lingala.zip4j.ZipFile
+  import net.lingala.zip4j.model.ZipParameters
+  import net.lingala.zip4j.model.enums.{CompressionLevel, CompressionMethod}
+  import java.nio.file.Paths
+
+  val pkgRoot = "io"
+  val testClassOutputPath = Paths.get("joern-cli", "frontends", "javasrc2cpg", "target", "scala-2.13", "test-classes")
+  val relativeTestCodePath = Paths.get(pkgRoot, "joern", "javasrc2cpg", "jartypereader", "testcode")
+
+  println("foo")
```

Comment:
```suggestion
```

Reply:
Whoops!

Reply:
Johannes probably left it there to test if you are good enough to spot it. Really well done @claudiuvursache 💯 🏅